<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculate</class>
 <widget class="QMainWindow" name="calculate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calculate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
  background-color: rgb(72,209,204);
  color: white; 
  border: 1px solid CadetBlue;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="answer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>97</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  background-color: rgb(175,238,238);
  color: Teal; 

}


background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>128</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pro">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>128</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dis">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>128</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>189</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>189</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>189</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>189</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>311</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>311</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>311</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>311</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exec">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>372</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>372</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>372</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>555</y>
      <width>244</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>history</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>433</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>433</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>BIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>494</y>
      <width>244</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>integral</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result_show">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>305</width>
      <height>92</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
  background-color: rgb(175,238,238);
  color: Teal; 
  border: 1px solid DarkCyan;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>128</y>
      <width>30</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>128</y>
      <width>30</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>98</y>
      <width>24</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  background-color: rgb(175,238,238);
  color: Teal; 

}


background-color : white;</string>
    </property>
    <property name="text">
     <string> =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>128</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>189</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ctan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>311</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>ctan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Y">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>372</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_V">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>433</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: Teal; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decr">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>494</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>555</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
